src/panels/developer-tools/statistics/dialog-statistics-adjust-sum.ts

```diff
@@ -63,6 +63,11 @@ export class DialogStatisticsFixUnsupportedUnitMetadata extends LitElement {
         @closed=${this.closeDialog}
         heading="Adjust sum for a specific time."
       >
+        <div class="card-content">
+          Adjust statistics if it incorrectly counted values that were already
+          counted. When adjusted, all subsequent values will be re-calculted
+          from the new value. This feature is limited to hourly statistics.
```

Comment:
It's not correct that only hourly statistics is updated, also 5-minute statistics is updated.
```suggestion
          When adjusted, the value at the starting point and all subsequent
          values will be adjusted with the given amount. This can for example
          be used if the sum is incorrect after a utility meter cycle reset.
```
Also, this should move to translations.

Reply:
But how would that work? How do you know what to apply to a 5 min and to a 60 min stat?